### ISSUE TITLE
FlexLayout widget fails to render in voila v0.4 and ipywidgets v8

### ISSUE THREAD

**Issue by nunupeke** [trungleduc/ipyflex#52]
In voila v0.4, I get the following Javascript error:
```
Model class 'FlexLayoutModel' from module 'ipyflex' is loaded but can not be instantiated
TypeError: this.widget_manager.display_model is not a function
    at d.initialize (https://cdn.jsdelivr.net/npm/ipyflex@%5E0.2.2/dist/index.js:75:782852)
    at new t.Model (http://localhost:8888/voila/templates/lab/static/voila.js?v=8db1cadbfba43a6dfc2d64128318e696bcb26b1783db1b991faf7404d438cd53200b6c434c4897064c090b68aa1151cd96c6dab50c3e138553365e6b30889172:61:579272)
    at new T (http://localhost:8888/voila/templates/lab/static/voila.js?v=8db1cadbfba43a6dfc2d64128318e696bcb26b1783db1b991faf7404d438cd53200b6c434c4897064c090b68aa1151cd96c6dab50c3e138553365e6b30889172:23:17621)
    at new S (http://localhost:8888/voila/templates/lab/static/voila.js?v=8db1cadbfba43a6dfc2d64128318e696bcb26b1783db1b991faf7404d438cd53200b6c434c4897064c090b68aa1151cd96c6dab50c3e138553365e6b30889172:23:23418)
    at new d (https://cdn.jsdelivr.net/npm/ipyflex@%5E0.2.2/dist/index.js:75:782350)
    at tn._make_model (http://localhost:8888/voila/templates/lab/static/voila.js?v=8db1cadbfba43a6dfc2d64128318e696bcb26b1783db1b991faf7404d438cd53200b6c434c4897064c090b68aa1151cd96c6dab50c3e138553365e6b30889172:458:112234)
```

In Jupyterlab, the `FlexLayout` widget renders fine, however.

**Comment by trungleduc** [comment]
Hi, I've just released `ipyflex` 0.2.6, can you update to the newest version and try again with Voila?

**Comment by nunupeke** [comment]
Works like a charm. Thank you very much!
